shuup/category_extensions/management/commands/update_populated_categories.py

```diff
@@ -0,0 +1,23 @@
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2016, Shoop Commerce Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from django.core.management import BaseCommand
+
+from shuup.category_extensions.models.category_populator import \
+    CategoryPopulator
+from shuup.core.models import ShopProduct
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        for shop_product in ShopProduct.objects.all():
```

Comment:
Imo we can't do this. Is there some filters we can use instead?

One approach would be to use `CategoryPopulator` in main loop.